data_structures/src/chain.rs

```diff
@@ -0,0 +1,27 @@
+#[derive(Debug)]
+pub struct ChainInfo {
+    environment: Environment,
+    consensus_constants: ConsensusConstants,
+    highest_block_checkpoint: Checkpoint
+}
+
+#[derive(Debug)]
+pub enum Environment {
+    Mainnet,
+    Testnet1
```

Comment:
This is not guaranteed to match the `enum Environment` in `ConfigManager`, but I don't think we can do much to avoid that.